cmd/osquery-extension/osquery-extension.go

```diff
@@ -6,15 +6,25 @@ import (
 	"os"
 	"os/signal"
 	"time"
+
+	"github.com/kolide/kit/version"
 )
 
 func main() {
-	flag.String("socket", "", "")
-	flag.Int("timeout", 0, "")
-	flag.Int("interval", 0, "")
-	flag.Bool("verbose", false, "")
+	var (
+		_         = flag.Bool("verbose", false, "")
```

Comment:
are these flags just defined to preserve compatibility with osquery's flag set? 

Mostly I'm trying to figure out why we define them if we don't use them.

Reply:
Dunno. I assume so.